src/localization/captions/Caption.js

```diff
@@ -78,8 +78,7 @@ export class Caption {
     }
 
     while (currentTime > this.lines[this.lineIndex].endTime) {
-      this.lineIndex++;
-      if (this.isFinished()) {
+      if (this.lineIndex++ ,this.isFinished()) {
```

Comment:
Neat, I didn't know that was a thing.

Reply:
Yeah it's more flashy than anything, but the last comma delimited item in parentheses is the thing that is returned.